### PR TITLE
vn/babel upgrade

### PR DESCRIPTION
**Pull request by vickingan**

### CODE REVIEW

File: es6-lib/upstream/core.js [hunk 1]
```diff
@@ -127,7 +125,7 @@ class Core extends CoreClient {
 
 
   getView(fourfour, onComplete, retried) {
-    const doGetView = () => this.getView(layer, onComplete, true);
+    const doGetView = () => this.getView(fourfour, onComplete, true);
```
Comment (sjzavaleta) [inline es6-lib/upstream/core.js:128]: This seems like a big change, but I don't know what the implications are

**Review by sjzavaleta — COMMENTED** [review]
I suppose the updated compiler required a ton of formatting changes. It was sort of hard to tell if you made any changes that weren't prompted by that.

File: docker/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 ## Building ##
 To build the image, run:
 ```
-cp -r ../lib ../scripts ../package.json ../npm-shrinkwrap.json .
+cp -r ../lib ../package.json .
```
[the remark this answers is not in the document]
Comment (vickingan) [reply docker/README.md:9]: It doesn't. I was testing docker as part of this upgrade and the README is out of date. There are no scripts and no npm-shrinkwrap in this repo.

File: es6-lib/config/index.js [hunk 3]
```diff
@@ -11,17 +11,15 @@
 import _ from 'underscore';
 import test from './test';
 import merge from 'deepmerge';
+import baseConfig from './config';
```
[the remark this answers is not in the document]
Comment (vickingan) [reply es6-lib/config/index.js:14]: There's no advantage, I'm trying to use as few requires as possible so we can eventually make the switch to [this](https://socrata.atlassian.net/browse/EN-47234).

File: es6-lib/upstream/core.js (hunk 1 above)
Comment (vickingan) [reply es6-lib/upstream/core.js:128]: This should be a typo. In the scope of this function, `layer` does not exist, only `fourfour`. This line is also calling `this.getView(...)` which refers to the current function.

**Comment by vickingan** [comment]
Updating babel doesn't require formatting changes. I added suggestions from jshint and cleaned up certain files as I was trying to use es6 modules with Node 14 earlier, so they are leftover remnants from [this](https://github.com/socrata-platform/geo-import/pull/113).

**Review by sjzavaleta — APPROVED** [review]

**Pull request merged**